src/App.js

```diff
@@ -1,6 +1,7 @@
 import "./App.css";
-
-import Homepage  from "./pages/Homepage/Homepage.js";
+import { Routes, Route } from "react-router-dom";
+// import Homepage  from "./pages/Homepage/Homepage.js";
+import {Homepage, ProductPage, Wishlist, Cart} from "./pages";
 
 import Navbar from "./components/Navbar/Navbar.js";
```

Comment:
use absolute path to import components.. will be much more redable.

Reply:
Sure I will add